docs/locales/en/LC_MESSAGES/deployment/networkrequirements.po

```diff
@@ -19,162 +19,162 @@ msgstr ""
 
 #: ../../deployment/networkrequirements.md:1
 msgid "网络要求"
-msgstr ""
+msgstr "Network Requirements"
 
 #: ../../deployment/networkrequirements.md:3
 msgid "前言"
-msgstr ""
+msgstr "Preface"
 
 #: ../../deployment/networkrequirements.md:5
 msgid "在部署的过程中，可能面临复杂的网络环境，一旦出现问题，需要花费许多时间去排查，特别是在引入了机构网关的情况。机构的网络拓扑可能是这样的："
-msgstr ""
+msgstr "During the deployment process, you may encounter a complex network environment. Once problems occur, it will take a lot of time to troubleshoot, especially when an institutional gateway is introduced. The network topology of an institution may look like this:"
 
 #: ../../deployment/networkrequirements.md:7
 msgid "包括 NAT 网关、防火墙、HTTP 代理服务器，也可能是多合一功能的设备，这些设备上可能有影响网络连通性的策略："
-msgstr ""
+msgstr "It includes NAT gateways, firewalls, and HTTP proxy servers. It may also be a device with integrated functions. There may be policies on these devices that affect network connectivity:"
 
 #: ../../deployment/networkrequirements.md:9
 msgid ""
 "NAT、防火墙可能会配置空闲链接保活时长，现象是如果链接上持续一段时间没有流量，链接会被关闭。表像为发送端发给改机构的报文被丢弃。从发送端 "
 "Envoy 上看触发 TCP 重传，从接收端 Envoy 报文直接被丢弃。"
-msgstr ""
+msgstr "NAT and firewalls may be configured with an idle connection keep - alive duration. The situation is that if there is no traffic on a connection for a certain period of time, the connection will be closed. The symptom is that the packets sent from the sender to the institution are discarded. From the perspective of the sender's Envoy, TCP retransmissions are triggered, and from the perspective of the receiver's Envoy, the packets are directly discarded. "
 
 #: ../../deployment/networkrequirements.md:10
 msgid "NAT、防火墙可能会配置 IP 白名单：如果没有配置白名单，可能会出现，TCP 握手请求直接被拒绝。"
-msgstr ""
+msgstr "NAT and firewalls may be configured with an IP whitelist. If the whitelist is not configured, the TCP handshake requests may be directly rejected."
 
 #: ../../deployment/networkrequirements.md:11
 msgid "如果 NAT、防火墙回复 Reset：从发送端看，HTTP 请求 503，TCP 链接被 Reset。"
-msgstr ""
+msgstr "If the NAT and firewall send back a \"Reset\" response: From the perspective of the sender, the HTTP request will result in a 503 status code, and the TCP connection will be reset.  "
```

Comment:
"If the NAT and firewall send back a \"Reset\" response  -> `"If the NAT and firewall send back a Reset response`